src/lib/pipelines/variableGroup.ts

```diff
@@ -267,9 +267,9 @@ export interface IVariablesMap {
  * @returns `IVariablesMap[]` with Varibale Group variables
  */
 export const buildVariablesMap = async (
-  variables: IVariablesMap[]
-): Promise<IVariablesMap> => {
-  const variablesMap: IVariablesMap = {};
+  variables: IVariableGroupDataVariable
```

Comment:
can you remove `IVariablesMap` interface declaration in this file?

Reply:
Good catch.